src/ImageIO/CGImageAnimation.cs

```diff
@@ -0,0 +1,141 @@
+//
+// ImageIO - CGImageAnimation.cs
+//
+// Authors:
+//	Whitney Schmidt  <whschm@microsoft.com>
+//
+// Copyright 2020, Microsoft Corp.
+//
+
+using System;
+using System.Runtime.InteropServices;
+using CoreFoundation;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+
+namespace ImageIO
+{
+
+    public partial class CGImageAnimation
+    {
+
+        public delegate void CGImageSourceAnimationBlock (nint index, CGImage image, out bool stop);
```

Comment:
Let's not use `Block` in our public API since it really does not mean much for a .NET developer, you can suffix stuff like this with `Handler` instead.

```suggestion
        public delegate void CGImageSourceAnimationHandler (nint index, CGImage image, out bool stop);
```